dropwizard-configuration/src/main/java/io/dropwizard/configuration/ResourceConfigurationSourceProvider.java

```diff
@@ -6,6 +6,11 @@
 public class ResourceConfigurationSourceProvider implements ConfigurationSourceProvider {
     @Override
     public InputStream open(String path) throws IOException {
+        InputStream result = getResourceAsStream(path);
+        return result == null && path.startsWith("/") ? getResourceAsStream(path.substring(1)) : result;
```

Comment:
You can add the Javadoc within your local `issue/1640` branch. When you commit and push the changes, the PR should update itself.

If you're feeling brave and want to keep the whole PR within a single commit, then you can [use `git rebase` to squash the two commits into one](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git).